src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -78,6 +79,44 @@ public void createPrimaryUserAffiliation(String eventPayload) {
     }
   }
 
+  @Override
+  @SneakyThrows
+  @Transactional
+  public void updatePrimaryUserAffiliation(String eventPayload) {
+    FolioExecutionContext currentContext = (FolioExecutionContext) folioExecutionContext.getInstance();
```

Comment:
no need to have this variable, setting context is working with a plain folioExecutionContext field

Reply:
Deleted